### PR TITLE
custom client of tm-load-test

### PR DESCRIPTION
**Pull request by puneetsingh166**
Added custom client in "customClient" directory.
Changed module name from "github.com/interchainio/tm-load-test" to "github.com/onomyprotocol/tm-load-test"

### CODE REVIEW

File: customClient/client.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package main
```
Comment (dzmitryhil) [inline customClient/client.go]: the code is not a go formatted. Format the code please and push changes. Also I enabled the linters on github workflow, so once you push the shares check the actions result.

File: Makefile [hunk 2]
```diff
@@ -38,3 +38,6 @@ lint: $(GOPATH)/bin/golangci-lint
 
 clean:
 	rm -rf $(BUILD_DIR)
+
+build-custom-client:
+	go build -o deploy/cosmos-load-test ./customClient/
```
Comment (dzmitryhil) [inline Makefile]: rename plz customClient -> onomy-load-test and move to cmd folder (cmd/onomy-load-test)

File: customClient/tx.go [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  encCfg := simapp.MakeTestEncodingConfig()
+  
+  // Create a new TxBuilder
+  txBuilder := encCfg.TxConfig.NewTxBuilder()
+  priv1, _, addr1 := testdata.KeyTestPubAddr()
+  _, _, addr2 := testdata.KeyTestPubAddr()
+  msg1 := banktypes.NewMsgSend(addr1, addr2, sdk.NewCoins(sdk.NewInt64Coin("footoken", 1)))
+  err := txBuilder.SetMsgs(msg1)
+  if err != nil {
+	  fmt.Println("error")
```
Comment (dzmitryhil) [inline customClient/tx.go]: What can be the reason for that error? And why you don't stop the execution if you get the error? The comments is the same for the cases where you check error and then just print it

Comment (dzmitryhil) [reply customClient/tx.go]: It is better to start returning the error and forward it in 

```
func (c *MyABCIAppClient) GenerateTx() ([]byte, error) {
	return GenTx() <-- this way
}
```

File: customClient/tx.go [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  }
+
+  txBuilder.SetGasLimit(200000)
+  txBuilder.SetFeeAmount(sdk.NewCoins(sdk.NewInt64Coin("footoken", 5)))
```
Comment (dzmitryhil) [inline customClient/tx.go]: please make constants for all string variables you use

**Review by dzmitryhil — CHANGES_REQUESTED** [review]

File: cmd/onomy-load-test/README.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
+# tm-load-test custom client for cosmos chain.
```
Comment (dzmitryhil) [inline cmd/onomy-load-test/README.md]: !! @puneetsingh166  Sorry for the late comment, but better move this "customClient" content to onomy tm-load-test, and add this repo as a dependency in gomod. as a result you will not have to copy constants on onomy and will be possible to update the test in onomy repo.  The comments abow still actual anyway !!!

**Review by dzmitryhil — CHANGES_REQUESTED** [review]

**Review by puneetsingh166 — COMMENTED** [review]
1. We have removed custom client from here and will move that in onomy repo.
2. This PR is coming with single change that is module name .using github.com/onomyprotocol/tm-load-test.

**Review by dzmitryhil — APPROVED** [review]

**Pull request merged**